Dockerfile

```diff
@@ -1,37 +1,39 @@
-FROM phusion/baseimage:0.9.19
+FROM phusion/baseimage:0.9.22
```

Comment:
Yeah ... I would like to stick to using a Ubuntu image instead of Debian or other linux. Is the raw ubuntu one very large?